R/teal_data-constructor.R

```diff
@@ -21,6 +21,9 @@
 #'
 #' @export
 #'
+#' @name teal_data
+#' @rdname teal_data
+#'
```

Comment:
is it necessary? `teal_data()` function has documentation under `?teal_data`

Reply:
Ah! Yes, this is not necessary, I think this is the remaining of some other tests that I did when I included `[.teal_data` in the docs of `teal_data`. Removed in here f74635a762876a39997a678b8e7fb85f1c857f88